tests/test_differential_abundance.py

```diff
@@ -69,6 +70,23 @@ def setUp(self):
         DA_fitZIG(self.tmp_otu_fp, self.tmp_otu_fp_fitZIG_out, self.tmp_map_fp, 'Individual', 'S1', 'S2')
         DA_DESeq2(self.tmp_otu_fp, self.tmp_otu_fp_DESeq2_out, self.tmp_map_fp, 'Individual', 'S1', 'S2', DESeq2_diagnostic_plots=False)
 
+
+    def test_check_mapping_file_category(self):
+        tmp_pmf, _ = parse_mapping_file_to_dict(self.tmp_map_fp)
+
+        with self.assertRaises(ValueError):
+            check_mapping_file_category(self.tmp_map_fp, 'Q', tmp_pmf, 'S1', 'S2')
```

Comment:
The tests are failing for me when I run them locally. These function calls need to be updated to match the new function signature.